lib/class-wp-rest-image-editor-controller.php

```diff
@@ -44,22 +44,22 @@ public function register_routes() {
 					'permission_callback' => array( $this, 'permission_callback' ),
 					'args'                => array(
 						'x'        => array(
-							'type'     => 'float',
+							'type'     => 'number',
```

Comment:
Strange, using `number` failed for me before.

Reply:
Hm, it seems to work, but what happens then? Is it rounded? Ideally we accept float values here.

Reply:
A `float` is referred to as a `number` in JSON Schema, https://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.4.2.1.

Reply:
More accurate link ( its the v4 spec which we use ) https://tools.ietf.org/html/draft-zyp-json-schema-04#section-3.5